tests/integration/test_agent.py

```diff
@@ -282,3 +282,15 @@ def test_browse_internet(http_server, current_test_name: str):
         assert 'OpenDevin is all you need!' in last_observation.content
     elif isinstance(last_observation, AgentDelegateObservation):
         assert 'OpenDevin is all you need!' in last_observation.outputs['content']
+
+
+def test_dummy_agent(current_test_name: str):
```

Comment:
You need to do ```@pytest.mark.skipif(os.environ.get('DEFAULT_AGENT') != 'DummyAgent')``` for this test

and also skip DummyAgent for every other test.

Reply:
how about separate `test_dummy_agent.py`?